frontend/lib/src/components/widgets/AudioInput/formatTime.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022-2024)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const formatTime = (timeMs: number): string => {
```

Comment:
**question:** Any particular reason we are writing our own formatter instead of leveraging something built in like `date.toLocaleTimeString('en-US', { minute: '2-digit', second: '2-digit' })`?

Now I know that this example includes a hardcoded `en-US`, but I suppose that brings up a follow-up question, should we consider supporting more locales out of the box? If we wanted to utilize `undefined` instead of `en-US`, we should get locale support for free.

Reply:
fantastic idea!